src/frontend/komponenter/Fagsak/Behandlingsresultat/Valutakurs/ValutakursTabellRad.tsx

```diff
@@ -47,6 +50,10 @@ const ValutakursTabellRad: React.FC<IProps> = ({
         barnIValutakurs: barn,
     });
 
+    if (erValutakursEkspandert && !skalRendreContentIEkspanderbartPanel) {
+        settSkalRendreContentIEkspanderbartPanel(true);
+    }
```

Comment:
Lurt å sørge for at valutakurstabellen ikke re-rendres, men kunne jo vært interessant å finne ut av årsaken til hvorfor den rerendres så mange ganger i utgangspunktet. Antar ihvertfall at det er det som er grunnen til at den er så "treg" 🤔

Reply:
Yes enig. Dette er jo egentlig bare en workaround